data/json/mapgen/basement/basement_lab_stairs.json

```diff
@@ -79,11 +79,11 @@
       "rotation": [ 0, 3 ],
       "rows": [
         "     |--------|         ",
-        "|----|XX.||..7|         ",
-        "|...i|X...D..>|         ",
-        "|&...|....D..>|         ",
-        "|....|...6|..7|         ",
-        "|--+---+----------------",
+        "|----|XX7|....|         ",
+        "|...i|X..D...>|         ",
+        "|&...|7..D...>|         ",
+        "|....|...|....|         ",
+        "|--+---DD6--------------",
```

Comment:
The way it is set up is so that the card reader can be reached from inside the lab, inside the airlock, and outside, and it opens both sets of doors.  My other idea was making two sets of doors and adding an ID card in the middle, but I figured this was an easier setup, as it changed less of the original design.

Reply:
aren't card readers generally on the right side ig?